app/code/Magento/InventoryShipping/Model/SourceSelectionInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryShipping\Model;
+
+interface SourceSelectionInterface
+{
+    /**
+     * Get item SKU
+     *
+     * @return string
+     */
+    public function getSku(): string;
+
+    /**
+     * Get source code
+     *
+     * @return string
+     */
+    public function getSourceCode(): string;
```

Comment:
This method is redundant.
We will have information about Source on more higher level